l2tscaffolder/lib/mapping_helper.py

```diff
@@ -100,21 +105,17 @@ def RenderTemplate(self, template_filename: str, context: dict) -> str:
 
   def SetDefaultPaths(self):
```

Comment:
Please refactor this, probably passing formatter_path and template_path to the ParserMapper initializer. There's no need to have multiple levels of initialization here, and SetTemplatePath and SetFormatterPath are only called by SetDefaultPaths (and the timesketch scaffolder, in this change).

Reply:
ok... so I changed this to take both formatter path and template path as part of the init function

and I also renamed the class to MapperHelper